packages/fab/index.js

```diff
@@ -22,13 +22,18 @@ export class Fab extends React.Component {
     });
   }
 
-  addIconClassToAllChildren = () => {
-    return React.Children.map(this.props.children, (item) => {
-      const className = `${item.props.className} mdc-fab__icon`;
-      const props = Object.assign({}, item.props, {className});
-      return React.cloneElement(item, props);
-    });
-  };
+  renderIcon() {
```

Comment:
comment for lines above^

You are not using this.state.classList. Please remove that from state.